### DISCUSSION TITLE
Force fields

### DISCUSSION THREAD

**Discussion by amerkoleci** [jrouwe/JoltPhysics#1310] [Q&A]
Hi 
This is feature request and not an issue, feel free to close it if not in your scope.

Would be possible to implement force fields like vortex, wind, turbulence etc?

Thanks,
Amer

**Comment by jrouwe** [comment]
I think this can be done using the current interface. You could use a sensor to detect bodies that are in a particular area, calculate the force you want to apply and call `BodyInterface::AddForce`.

**Reply by amerkoleci** [reply]
Nice, 
Would be possible to have some small example? Just to make sure i don't write wrong code 😀

**Reply by Mahmoud1205** [reply]
The 'General > Sensor' sample can be tweaked to be a vortex effect. Wind can be implemented by applying a force to bodies in the wind direction.
